src/main/java/frc/robot/subsystems/AgitatorSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.config.Config;
+import frc.robot.config.FluidConstant;
+
+public class Agitator extends SubsystemBase {
+
+    private VictorSPX agitatorMotor;
+
+//Variables:
+private final double AGITATOR_SPEED = 1.0;
```

Comment:
As a general rule, you should indent your code properly using 4 space indentation. I believe there is a VS Code plugin called Beautify File that could help you do this, though I don't know how well it works with java files.